sys-boot/plymouth/plymouth-0.9.5-r1.ebuild

```diff
@@ -72,7 +72,7 @@ src_configure() {
 		$(use_enable static-libs static)
 		$(use_enable debug tracing)
 		$(use_enable gtk gtk)
-		$(use_enable libkms drm)
+		$(use_enable drm drm)
```

Comment:
`$(use_enable drm)`

Reply:
I've fixed that, reordered and also sorted the double gtk gtk too